server/scripts/get-email-list.js

```diff
@@ -17,8 +17,8 @@ require('dotenv').config({ path: '../.env' });
             OR CAST(p.body->> 'interested' AS TEXT) IS NULL)
         AND (
             to_timestamp(p.body->>'userUpdatedAt', 'YYYY-MM-DD') < CURRENT_TIMESTAMP - interval '6 weeks'
-        )`
-  );
+            to_timestamp(p.body->>'userUpdatedAt', 'YYYY-MM-DD') < CURRENT_TIMESTAMP - interval '6 weeks'
+        )`);
```

Comment:
I accidentally touched this file and can't not edit it because of the pre-commit hook, shouldn't change anything